README.md

```diff
@@ -118,7 +124,115 @@ things like new user registration, and those settings are also configured within
  `politeiawww.conf`. The current code should work with most SSL-based SMTP servers
 (but not TLS) using username and password as authentication.
 
-#### 4. Build the programs:
+#### 4. Setup Cockroachdb:
```

Comment:
We may want to call this cockroach or postgres since they use the same driver. Might as well.

Reply:
Fixed.